contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,105 @@
+<frontmatter>
+  title: Introduction to iOS App Development
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to iOS Development
+
+Authors: [Bryan Lew](https://github.com/blewjy)
+
+## 1. Overview: What is iOS?
+
+iOS is the mobile operating system that runs on Apple's mobile devices, most notably the iPhone and the iPad. Programs or applications that run on iOS can be downloaded officially from the App Store, and developers of iOS applications can submit their own applications to the App Store to share it with the rest of the world. 
```

Comment:
Does this require a paid license?

Reply:
Yes, it does. However i dont think the overview is a good place to mention this... I added this information to the section that talks about app store submission.